### PR TITLE
add succes modal

### PR DESCRIPTION
**Pull request by IgboPharaoh**
- Add success modal after user install fresheyes bot

### CODE REVIEW

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  useEffect(() => {
+    const isInstalled = params.get("installation_id") !== null;
+    if (isInstalled) {
+      setIsOpen(true);
+    }
+  }, [pathname]);
+
+  const removeModal = () => {
+    setIsOpen(false);
+    let url = new URL(window.location.href);
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:20]: We can use the `useRouter` from `next/navigation` `router.push` here. It saves us from depending on the window object which throws a warning in the console if undefined. We also remove one line of code

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const params = useSearchParams();
+
+  useEffect(() => {
+    const isInstalled = params.get("installation_id") !== null;
+    if (isInstalled) {
+      setIsOpen(true);
+    }
+  }, [pathname]);
+
+  const removeModal = () => {
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:18]:
```suggestion
  const closeModal = () => {
```
nit: I think closeMoal is more relatable as a variable name

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    setIsOpen(false);
+    let url = new URL(window.location.href);
+    history.pushState({}, "", url.href);
+  };
+
+  return (
+    <>
+      {isOpen && (
+        <div
+          className='fixed top-0 bottom-0 right-0 left-0 flex items-center justify-center dark:from-inherit bg-zinc-800/30 backdrop-blur-sm z-30 p-3 cursor-'
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:28]: Did you want to write `cursor-pointer`?

```suggestion
          className='fixed top-0 bottom-0 right-0 left-0 flex items-center justify-center dark:from-inherit bg-zinc-800/30 backdrop-blur-sm z-30 p-3 cursor-pointer'
```

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+          onClick={removeModal}
+        >
+          <div className='rounded-md min-h-[300px] max-w-[500px] w-full bg-gray-100 dark:bg-black z-50 border-[0.25px] border-white flex items-center justify-center flex-col p-3 md:p-6 gap-6 relative'>
+            <section className=' relative h-32 w-32'>
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:32]:
```suggestion
            <section className='relative h-32 w-32'>
```

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+              <Image src={checkmarkIcon} alt='checkmark icon' fill className=' w-full h-full' />
+            </section>
+            <section className=' text-center'>
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:35]:
```suggestion
            <section className='text-center'>
```

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+              <h1 className=' font-bold text-3xl'>Success !</h1>
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:36]:
```suggestion
              <h1 className='font-bold text-3xl'>Success !</h1>
```

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <>
+      {isOpen && (
+        <div
+          className='fixed top-0 bottom-0 right-0 left-0 flex items-center justify-center dark:from-inherit bg-zinc-800/30 backdrop-blur-sm z-30 p-3 cursor-'
+          onClick={removeModal}
+        >
+          <div className='rounded-md min-h-[300px] max-w-[500px] w-full bg-gray-100 dark:bg-black z-50 border-[0.25px] border-white flex items-center justify-center flex-col p-3 md:p-6 gap-6 relative'>
+            <section className=' relative h-32 w-32'>
+              <Image src={checkmarkIcon} alt='checkmark icon' fill className=' w-full h-full' />
```
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:33]:
```suggestion
              <Image src={checkmarkIcon} alt='checkmark icon' fill className='w-full h-full' />
```

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx (hunk 6 above)
Comment (Extheoisah) [inline client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:36]:
```suggestion
              <h1 className=' font-bold text-3xl'>Success!</h1>
```

**Review by Extheoisah — CHANGES_REQUESTED** [review]
Looks good. Just a few nits!

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx (hunk 1 above)
Comment (IgboPharaoh) [reply client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:20]: Alright, will make changes using `useRouter`

File: client/fresheyes-web/src/app/components/modal/SuccessModal.tsx (hunk 6 above)
Comment (IgboPharaoh) [reply client/fresheyes-web/src/app/components/modal/SuccessModal.tsx:36]: will make necessary corrections for changes requested

**Comment by IgboPharaoh** [comment]
will make necessary corrections for changes requested

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**